### PR TITLE
Combining productSearch query

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) combines the productSearch query in the Catalog Service with the same query in Live Search. [MSRCH-4507](https://jira.corp.adobe.com/browse/MSRCH-4507) and [MSRCH-4455](https://jira.corp.adobe.com/browse/MSRCH-4455).

## Affected pages

- https://developer.adobe.com/commerce/services/graphql/catalog-service/product-search/
- https://developer.adobe.com/commerce/services/graphql/live-search/product-search/

### CODE REVIEW

**Review by vsvd — APPROVED** [review]

File: src/pages/graphql/live-search/product-search.md [hunk 1]
```diff
@@ -373,23 +387,35 @@ The query response can also contain the following top-level fields and objects:
...
+--- | ---
+`Magento-Customer-Group` | (**Catalog Service Only**) This value is available in the `customer_group_data_exporter` database table.
+`Magento-Environment-Id` | This value is displayed at **Stores** > **Configuration** > **Services** > **Magento Services** > **SaaS Environment** or can be obtained by running the `bin/magento config:show services_connector/services_id/environment_id` command.
+`Magento-Store-Code` | The code assigned to the store associated with the active store view. For example, `main_website_store`.
+`Magento-Store-View-Code` | The code assigned to the active store view. For example, `default`.
+`Magento-Website-Code` | The code assigned to the website associated with the active store view. For example, `base`.
+`X-Api-Key` | This value must be set to `search_gql`.
 
-## Example usage
+### Live Search
```
Comment (keharper) [inline src/pages/graphql/live-search/product-search.md:414]: You're missing the Example usage H2.

File: src/pages/graphql/live-search/product-search.md [hunk 2]
```diff
@@ -910,7 +1185,7 @@ Field | Data Type | Description
 
 ## Output fields
 
-The `AttributeMetadataResponse` return object can contain the following fields:
+The `productSearchResponse` return object can contain the following fields:
```
Comment (keharper) [inline src/pages/graphql/live-search/product-search.md:1188]: Move the Common fields h3 here

File: src/pages/graphql/live-search/product-search.md [hunk 3]
```diff
@@ -373,23 +390,39 @@ The query response can also contain the following top-level fields and objects:
 - `suggestions` - An array of strings that include the names of products and categories that exist in the catalog that are similar to the search query.
 - `total_count` - The number of products returned.
 
-## Endpoint
+## Endpoints
```
Comment (keharper) [inline src/pages/graphql/live-search/product-search.md:393]: We no longer need the InlineAlert on lines 379-381

File: src/pages/graphql/live-search/product-search.md [hunk 4]
```diff
@@ -910,7 +1192,27 @@ Field | Data Type | Description
 
 ## Output fields
 
-The `AttributeMetadataResponse` return object can contain the following fields:
+The `productSearchResponse` return object can contain the following fields:
+
+### Common fields
+
+#### ProductSearchItem data type
```
Comment (keharper) [inline src/pages/graphql/live-search/product-search.md:1199]: Lines 1199 - 1215 are in the wrong place.

**Review by keharper — APPROVED** [review]

**Pull request merged**